src/components/FileTreeNode/index.spec.tsx

```diff
@@ -732,7 +736,7 @@ describe(__filename, () => {
     const externalVersion = fakeVersion;
     const nodeId = externalVersion.file.selected_file;
 
-    const store = createStoreWithVersion(externalVersion);
+    const store = createStoreWithVersion({ makeCurrent: true });
```

Comment:
Same here. I'm commenting on each one just so we don't miss any in the next review.